hypertrace-view-generator/helm/values.yaml

```diff
@@ -204,63 +110,52 @@ spanEventView:
       requests:
         cpu: 0.1
         memory: 384Mi
-  generator:
-    deploymentName: span-event-view-generator
-    podLabels:
-      app: span-event-view-generator
-    # Careful with deployment selector match labels. You can append labels but not remove them once deployed.
-    # Use the pod labels above for custom pod labels
-    deploymentSelectorMatchLabels:
-      app: span-event-view-generator
-    javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
-    livenessProbe:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-    readinessProbe:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-    resources:
-      requests:
-        cpu: 0.1
-        memory: 320Mi
-      limits:
-        cpu: 0.2
-        memory: 512Mi
 
-mergeAllViews: false
+############################################
+# View Generation Deployments              #
+############################################
 
-allViews:
-  creator:
-    jobName: all-views-creation-job
-    javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
-    resources:
-      limits:
-        cpu: 1.0
-        memory: 384Mi
-      requests:
-        cpu: 0.1
-        memory: 384Mi
-  generator:
-    deploymentName: all-views-generator
-    podLabels:
-      app: all-views-generator
-    deploymentSelectorMatchLabels:
-      app: all-views-generator
-    javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
-    livenessProbe:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-    readinessProbe:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-    resources:
-      limits:
-        cpu: 1.0
-        memory: 512Mi
-      requests:
-        cpu: 0.2
-        memory: 320Mi
+viewGeneratorGroups:
+  - name: all-views-generator
```

Comment:
instead of array, use a map. overriding values is much easier with map.

Reply:
done